var/spack/repos/builtin/packages/akantu/package.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Akantu(CMakePackage):
+    """
+    Akantu means a little element in Kinyarwanda, a Bantu language. From now
+    on it is also an opensource object-oriented Finite Element library which
+    has the ambition to be generic and efficient.
+
+    """
+    homepage = "https://akantu.ch"
+    url      = "https://gitlab.com/akantu/akantu/-/archive/master/akantu-master.tar.bz2"
+
+    maintainers = ['nrichart']
+
+    version('master', git='https://gitlab.com/akantu/akantu.git')
```

Comment:
I see there are tags in the Gitlab repo. Is it worth adding a tagged release here?